exporter/constants.py

```diff
@@ -39,3 +39,12 @@
     MID_POLY_TYPE: "OBJ",
     HIGH_POLY_TYPE: "OBJ",
 }
+
+# Export type to format mappings (glTF)
+# TODO: find a more modular solution
+EXPORT_FILE_TYPES_GLTF = {
+    STATIC_MESH_TYPE: "GLB",
```

Comment:
If the Preferences setting for GLTF was an enum type instead of a bool, we could then just keep the single declaration of `EXPORT_FILE_TYPES` but instead of hard-coding the extension, we could just get the value of the enum (e.g. `FBX`, `GLTF`, `GLB`, etc.) - I think this would make it easier to extend and reduce code duplication.